### PR TITLE
PoC of exporting image  using docker pull/push

### PR DESCRIPTION
**Pull request by kadel**
PoC for #2 

Requires exposed Internal OpenShift registry to be exposed https://docs.openshift.org/latest/install_config/install/docker_registry.html#exposing-the-registry

This is trying to do pull/push aproach via machine that is running export

- [x] Pull images from OpenShift registry
- [x] update artifacts
- [x] Push images to other registry
- [x] Document this in readme
- [x] explain `internal` and `all` options for `--export-images`

### CODE REVIEW

**Comment by kadel** [comment]
Hey @surajssd, if you have moment, can you please check this? I would like to know your opinion on this.

File: openshift2nulecule/cli/main.py [hunk 1]
```diff
@@ -46,19 +46,52 @@ def __init__(self):
...
                                  action='store_true')
 
+        self.parser.add_argument("--oc-registry-host",
+                                 help="Hostname of exposed internal OpenShift registry",
+                                 required=False)
+
+        self.parser.add_argument("--export-images",
+                                 help="Pull images from OpenShift registry to "
+                                      "local Docker instance and push them to "
+                                      "remote registry (--registry-host) ",
```
Comment (surajssd) [inline openshift2nulecule/cli/main.py]: The help is misleading, maybe explaining what `internal` and `all` may help.

File: openshift2nulecule/cli/main.py [hunk 2]
```diff
@@ -46,19 +46,52 @@ def __init__(self):
...
+                                 choices=["none", "internal", "all"],
+                                 default="none",
+                                 required=False)
+
+        self.parser.add_argument("--registry-host",
+                                 help="External registry hostname",
```
Comment (surajssd) [inline openshift2nulecule/cli/main.py]: hostname or IP address, what if this is not specified? Does this assumes that the `localhost` is taken by default?

File: openshift2nulecule/cli/main.py (hunk 1 above)
Comment (kadel) [inline openshift2nulecule/cli/main.py]: You are right, I will try to explain this better.

File: openshift2nulecule/cli/main.py (hunk 2 above)
Comment (kadel) [inline openshift2nulecule/cli/main.py]: If `--registry-host` is not specified than images are only pulled by Docker, but won't be pushed to any registry. 

Right now this is always optional even with `--export-images`. Now when I'm thinking about this, maybe this should be required when `--export-images`!=none. 
I can't think of use case when someone would only want to pull images and not to push them somewhere.

**Pull request merged**